tests/functional/pod_and_daemons/test_odf_resources_requests_and_limits.py

```
@@ -12,36 +12,44 @@
-        """
+
+        # Guideline: Use logger.test_step() for workflow phases  (no manual formatting/numbering)
+        logger.test_step("Identify an available operational ODF worker node")
+        worker_nodes = get_worker_nodes()
+        if not worker_nodes:
+            pytest.fail(
+                "No worker nodes found in the cluster to execute the drain operation."
+            )
+
+        target_node = worker_nodes[0]
+        logger.info(f"Target worker node selected for eviction: {target_node}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)